packages/legacy-workbench/src/js/angular/guides/steps/core/plugin.js

```diff
@@ -103,6 +110,51 @@ PluginRegistry.add('guide.step', [
             return stepDescription;
         }
     },
+    {
+        guideBlockName: 'copy-text-element',
+        getSteps: (options, services) => {
+            const $translate = services.$translate;
+            const GuideUtils = services.GuideUtils;
+
+            const text = options.text;
+            const code = document.createElement('code');
+            code.innerText = text;
+
+            const copy = document.createElement('button');
+            const copyToEditorButtonClass = 'guide-copy-to-editor-query-button';
+            copy.className = `btn btn-sm btn-secondary ${copyToEditorButtonClass}`;
```

Comment:
The class name should not be the same as the one used in the sparql-editor step.